### ISSUE TITLE
SLF4J warnings after updating validator to 1.0.8.0

### ISSUE THREAD

**Issue by YaroslavBurlakaLS** [networknt/json-schema-validator#719]
Have following warnings in console:

SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.
SLF4J: Ignoring binding found at [jar:file:/C:/Users/*******/.m2/repository/org/slf4j/slf4j-log4j12/1.7.10/slf4j-log4j12-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.

Can this be fixed from validator itself, or I need to follow the intructions of SLF4J ?

**Comment by stevehu** [comment]
I don't know how you get this error. We recently upgraded it to 2.0.7, which might cause some issues if you include a lower version in your application. Please give us more context info. 

https://github.com/networknt/json-schema-validator/blob/master/pom.xml#L87

**Comment by YaroslavBurlakaLS** [comment]
Well, my application does not use SLF4J at all, atleast explicitly (I can't find anything in my project by searching for 'SLF4J' keyword)
This warning just appears at the start of application

**Comment by fdutton** [comment]
@YaroslavBurlakaLS There are many libraries in Maven Central that use SLF4J so you are likely pulling it in through one of those. Try updating your POM to exclude SLF4J.
```
    <dependency>
      <groupId>com.networknt</groupId>
      <artifactId>json-schema-validator</artifactId>
      <version>1.0.80</version>
      <exclusions>
        <exclusion>
          <groupId>org.slf4j</groupId>
          <artifactId>slf4j-api</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

**Comment by YaroslavBurlakaLS** [comment]
> 

this fixed my issue, so I guess itcan be accepted as a soulution

**Comment by stevehu** [comment]
Thanks @fdutton to provide a solution and the quick confirmation from @YaroslavBurlakaLS
